maven-plugin/src/main/java/sortpom/VerifyMojo.java

```diff
@@ -93,6 +93,12 @@
     private String lineSeparator;
 
     /**
+     * Ignore line separators when comparing current POM with sorted one
+     */
+    @Parameter(property = "sort.ignoreLineSeparators", defaultValue = "true")
```

Comment:
Question: The verify goal always compare only the structure to determine if the verification fails or not. (SortPomImp.isPomElementsSorted())
Do we need to change that if we have this parameter?
Otherwise this parameter should not be present for the verify mojo.